packages/image_picker/lib/image_picker.dart

```diff
@@ -74,16 +74,18 @@ class ImagePicker {
   /// The [source] argument controls where the video comes from. This can
   /// be either [ImageSource.camera] or [ImageSource.gallery].
   ///
+  /// The [maxDuration] argument specifies the maximum duration of the captured video.
```

Comment:
explain when [maxDuration] is null, the maxDuration is infinite

Reply:
Sorted